### PR TITLE
got rid of "pattern matching", fix spacing, added contents

### PR DESCRIPTION
**Pull request by NguyenTran88**

### CODE REVIEW

File: recitation_1/bash.md [hunk 1]
```diff
@@ -136,22 +147,21 @@ $cut -d "delimiter" -f (field number) file.txt
...
+$ Tran
 ```
-### Basic pattern matching:
-TBA
 
 Exercise:
 ------
 ### Agenda:
-- First, we will go through Dr.B's "grade.sh" script in class. We will apply what we just learnt above to see what and why the code is doing what it does. You can obtain a copy of the code on Coursework.
+- First, we will go through Dr.B's "grade.sh" script in class. We will apply what we just learnt above to see what and why the code is doing what it does. A copy of the code should be available on Coursework.
```
Comment (maturanamateo) [inline recitation_1/bash.md:164]:
```suggestion
- First, we will go through Dr.B's "grade.sh" script in class. We will apply what we just learnt above to see what and why the code is doing what it does. A copy of the code should be available on Courseworks.

```

Comment (maturanamateo) [reply recitation_1/bash.md:164]: @NguyenTran88 you should also ask Dr B if you can put the code into the recitation 1 folder. Would be best practice to have it all in one location. Any code we refer to in recitation should ideally be in the repo.

**Review by maturanamateo — CHANGES_REQUESTED** [review]

File: recitation_1/grade (1).sh [hunk 2]
```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
```
Comment (xurxo-riesco) [inline recitation_1/grade (1).sh:1]: can we rename this file to just grade.sh?

**Review by xurxo-riesco — COMMENTED** [review]
can u also add acknowledgments and the bottom, something along the lines of the one in git.md

File: recitation_1/grade (1).sh (hunk 2 above)
Comment (NguyenTran88) [reply recitation_1/grade (1).sh]: Done !

File: recitation_1/bash.md [hunk 3]
```diff
@@ -177,6 +196,10 @@ echo "sum[$1..$2] = $return_val"
 
 Based on the script above, `$./sum.sh 1 5` will have an output of `15` 
 
+Acknowledgement
+------
+This was developed by TAs Nguyen Tran, Xurxo Riesco, and Mateo Maturana for COMS 3157 Spring '23
```
Comment (xurxo-riesco) [inline recitation_1/bash.md:201]: Just you is fine, we are not counting reviewers as contributors

Comment (NguyenTran88) [reply recitation_1/bash.md]: done !

**Review by maturanamateo — APPROVED** [review]
lgtm

**Pull request merged**